models/actions/runner.go

```diff
@@ -202,6 +202,8 @@ func (opts FindRunnerOptions) ToConds() builder.Cond {
 			c = c.Or(builder.Eq{"repo_id": 0, "owner_id": 0})
 		}
 		cond = cond.And(c)
+	} else if !opts.WithAvailable {
+		cond = cond.And(builder.Eq{"repo_id": 0, "owner_id": 0})
```

Comment:
Don't quite understand what it means ...

What if a "global runner" has `repo_id=0 AND owner_id=0`?